dev-workflow-ui/src/ch/ivyteam/workflowui/notification/NotificationActionDTO.java

```diff
@@ -1,14 +1,10 @@
 package ch.ivyteam.workflowui.notification;
 
 import ch.ivyteam.ivy.model.value.WebLink;
-import ch.ivyteam.ivy.notification.web.WebNotificationAction;
 
-public record NotificationActionDTO(WebNotificationAction.Type type, WebLink link) {
+public record NotificationActionDTO(WebLink link, String title) {
```

Comment:
Instead of using a record that contains the same things as NotificationAction I would implement a real java bean here so that you can access it from xhtml with attributes like notification.runAction.title instead of notification.runAction.title(). In this way it brings some value. Otherwise you can directly use NotificationAction here.